src/helm/benchmark/run_specs/builtin_run_specs.py

```diff
@@ -0,0 +1,1258 @@
+from typing import Any, Dict, List, Optional, Set
+
+from helm.benchmark.adaptation.adapter_spec import (
+    ADAPT_GENERATION,
+    ADAPT_MULTIPLE_CHOICE_JOINT,
+    ADAPT_MULTIPLE_CHOICE_SEPARATE_ORIGINAL,
+    ADAPT_RANKING_BINARY,
+    AdapterSpec,
+)
+from helm.benchmark.adaptation.adapters.binary_ranking_adapter import BinaryRankingAdapter
+from helm.benchmark.adaptation.common_adapter_specs import (
+    get_completion_adapter_spec,
+    get_generation_adapter_spec,
+    get_language_modeling_adapter_spec,
+    get_machine_translation_adapter_spec,
+    get_multiple_choice_adapter_spec,
+    get_ranking_binary_adapter_spec,
+    get_summarization_adapter_spec,
+)
+from helm.benchmark.metrics.common_metric_specs import (
+    get_basic_metric_specs,
+    get_bias_metric_specs,
+    get_classification_metric_specs,
+    get_copyright_metric_specs,
+    get_disinformation_metric_specs,
+    get_exact_match_metric_specs,
+    get_f1_metric_specs,
+    get_generative_harms_metric_specs,
+    get_language_modeling_metric_specs,
+    get_numeracy_metric_specs,
+    get_open_ended_generation_metric_specs,
+    get_summarization_metric_specs,
+)
+from helm.benchmark.metrics.metric import MetricSpec
+from helm.benchmark.run_spec import RunSpec, run_spec_function
+from helm.benchmark.runner import get_benchmark_output_path
+from helm.benchmark.scenarios.scenario import ScenarioSpec, get_scenario_cache_path
+from helm.common.hierarchical_logger import hlog, htrack
+
+
+############################################################
```

Comment:
Why is this file called `builtin`?  This is mostly HELM classic stuff, right?

Reply:
Other alternatives: `basic`, `common`, `classic`, `lite` (the latter two means more splitting)

Reply:
I think classic and lite should be separate.  I'd be okay duplicating run specs too...  What's the diff between basic and common?

Reply:
I went with this split for now: `classic_run_specs.py`, `lite_run_specs.py`, `cleva_run_specs.py`, `decodingtrust_run_specs.py`, `instruction_following_run_specs.py`

Reply:
I'll move the stuff that is not actually used in HELM Classic out of `classic_run_specs.py` eventually. Filed #2388 to track.